templates/distribution/scripts/apply.sh.tpl

```diff
@@ -9,6 +9,14 @@ vendorPath="{{ .paths.vendorPath }}"
 
 $kustomizebin build --load_restrictor LoadRestrictionsNone . > out.yaml
 
+{{- if and (index .spec.distribution.common "customRegistry") (ne .spec.distribution.common.customRegistry "") }}
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  sed -i "" 's#registry.sighup.io/fury#{{.spec.distribution.common.customRegistry}}#g' out.yaml
+else
+  sed -i 's#registry.sighup.io/fury#{{.spec.distribution.common.customRegistry}}#g' out.yaml
```

Comment:
Two "side effects" I noticed testing:

1. This will patch also the images in the plugins, if this is the intended behaviour we should document it.
2. When you are using the `customPatches.images` config, for example to change an image name, the sed replace will be also applied afterwards "overriding" what you put as `newImage` in the patch, if the image is coming from registry.sighup.io. Example: you want to use `registry.sighup.io/fury/grafana/grafana-patched` (does not exists) instead of `registry.sighup.io/fury/grafana/grafana`. The registry part will be replaced when using the `customRegistry` part. If this is the intended behaviour, this should be documented too.